### PR TITLE
icon: provide app icon

### PR DESCRIPTION
**Pull request by jimmac**
- based on the new logo
- includes a symbolic variant

### CODE REVIEW

File: data/input-tablet-missing-symbolic.svg [hunk 1]
```diff
@@ -1,86 +1 @@
...
...xt-align:start;text-decoration-line:none;text-decoration-style:solid;text-decoration-color:#000;text-transform:none;text-orientation:mixed;shape-padding:0;isolation:auto;mix-blend-mode:normal" overflow="visible" fill-rule="evenodd"/><path d="M8 0v1.5a.506.506 0 0 1-.147.356.484.484 0 0 1-.345.144H4.496a1.498 1.498 0 0 0-1.297 2.248c.266.463.761.752 1.295.752h3.01c.18 0 .34.092.43.248A.506.506 0 0 1 8 5.5V6h1v-.5A1.498 1.498 0 0 0 7.504 4h-3.01a.491.491 0 0 1-.428-.248.507.507 0 0 1 0-.504.488.488 0 0 1 .43-.248h3.012A1.501 1.501 0 0 0 9 1.5V0z" style="line-height:normal;font-variant-ligatures:normal;font-variant-position:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-alternates:normal;font-feature-settings:normal;text-indent:0;text-align:start;text-decoration-line:none;text-decoration-style:solid;text-decoration-color:#000;text-transform:none;text-orientation:mixed;shape-padding:0;isolation:auto;mix-blend-mode:normal" overflow="visible"/></g><path overflow="visible" style="marker:none" d="M11 11h1.375l1.125 1.094L14.594 11H16v1.469l-1.094 1.062L16 14.594V16h-1.438L13.5 14.937 12.437 16H11v-1.406l1.063-1.063L11 12.47z" color="#bebebe" fill="#474747"/></svg>
```
Comment (whot) [inline data/input-tablet-missing-symbolic.svg:1]: Did you change this file or is it just compressed SVG? I can't see any difference between the files but there may be a moved pixel or two.

**Review by whot — APPROVED** [review]
Looks great, thanks!

Just waiting for a comment regarding the "missing tablet" icon, depending on that I'll split out that hunk or not.

File: data/input-tablet-missing-symbolic.svg (hunk 1 above)
Comment (jimmac) [reply data/input-tablet-missing-symbolic.svg:1]: Sorry that was unintended. Yea I run `svgo` on all my exported inkscape SVGs and must have been lazy with a glob match.

Comment (whot) [reply data/input-tablet-missing-symbolic.svg:1]: thx, I'll split this out then

**Comment by whot** [comment]
Merged as #176, thanks

**Pull request closed without merging**